src/components/Filters/Filters.tsx

```diff
@@ -97,20 +102,27 @@ const Filters: React.FC<FiltersProps> = ({
   }, [filtersValues, fields]);
 
   useEffect(() => {
```

Comment:
Filters `useEffect`s pretty grew up by last updates including error handling and request cancelation. With adding another logic I would move them to testable custom hook.